docs_user/modules/assembly_ospdo-scale-down-pre-database-adoption.adoc

```diff
@@ -0,0 +1,215 @@
+[id="ospdo_scale_down_pre_database_adoption_{context}"]
+
+= OSPdO scale down pre database adoption
+
+This section scales down and removes OSPdO resources in favor of the RHOSO ones.
+
+.Procedure
+
+. Scale-down RHOSO OpenStack Operator controller-manager to 0 replicas and delete its OpenStackControlPlane’s OpenStackClient pod temporarily, so that we can use the OSPdO controller-manager to clean up some of its resources, needed because OSPdO ultimately can’t act due to a pod name collision between its OpenStackClient and RHOSO’s OpenStackClient.
++
+----
+oc patch csv -n openstack-operators openstack-operator.v0.0.1 --type json   -p="[{"op": "replace", "path": "/spec/install/spec/deployments/0/spec/replicas", "value": "0"}]"
+oc delete openstackclients.client.openstack.org --all
+----
++
+. Delete the OSPdO OpenStackControlPlane
++
+----
+oc delete openstackcontrolplanes.osp-director.openstack.org -n openstack --all
+----
++
+. Delete the OSPdO OpenStackNetConfig (this will remove OSPdO’s associated NNCPs)
++
+----
+oc delete osnetconfig -n openstack --all
+----
++
+. Label the third OCP node, the once which is holding the OSPdO vm (the one that was not labeled earlier)
++
+----
+oc label nodes <ostest-master-node> type=openstack
+----
++
```

Comment:
```suggestion
. Label the {OpenShiftShort} node that contains the OSPdO virtual machine (VM):
+
----
$ oc label nodes <ostest-master-node> type=openstack
----
+
* Replace ` <ostest-master-node>` with the remaining master node that contains the OSPdO VM.
```
@pinikomarov Please ensure that this rewrite is accurate.

Reply:
ok, that's accurate,
ostest was an arbitrary name chosen for ospdo deployments , no special meaning, I'll change it to something general: ospdo_vm_master_node